src/lib/constants.js

```diff
@@ -0,0 +1,27 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+module.exports = {
```

Comment:
Ah OK - no I hadn't noticed that dir, I'll move it in there. I made the file to get `BADNUM` out of `src/plots/cartesian/constants.js`, perhaps eventually this should move there too.